src/app/views/users/_user.html.erb

```diff
@@ -15,8 +15,9 @@
             @<%= user.username %>
           <% end %>
         </p>
-
-        <%= render "users/follow", current_user: current_user, user: user %>
+        <% if show_actions?  %>
+          <%= render "users/actions", user: user %>
```

Comment:
we can render this file outside of the user partial, in the comments/index and other pages that render the user partial

Reply:
it will break the design !